service/dap/server.go

```diff
@@ -2061,10 +2061,7 @@ func (s *Server) onEvaluateRequest(request *dap.EvaluateRequest) {
 		}
 		var opts convertVariableFlags
 		if ctxt == "clipboard" || ctxt == "variables" {
```

Comment:
Let's add a comment that these contexts are usually used to copy the value, which is why we want it in full.

Reply:
I'll wait to merge until this is addressed.